requests/utils.py

```diff
@@ -693,6 +693,17 @@ def to_native_string(string, encoding='ascii'):
     return out
 
 
+def convert_to_string(obj):
+    """
+    Given a non-string object (neither unicode, nor bytes), returns a string
+    representation of that object.
+    """
```

Comment:
This docstring isn't quite right. It should be:

```
Given an object, converts it to a string. If the object is already an instance of `basestring`,
this is a no-op. Otherwise, uses the string representation of that object.
```